### ISSUE TITLE
`Extract variable` feature does not work

### ISSUE THREAD

**Issue by AndreiOrmanji** [deakjahn/vscode-php-allfactor#5]
Steps to reproduce:
1. Select range from screenshot ![image](https://user-images.githubusercontent.com/48985454/230559194-38c55cd4-afaf-4560-9464-714388415224.png)
2. Choose "Extract variable"

Actual result:
1. No changes in code.
2. Notification received 
![image](https://user-images.githubusercontent.com/48985454/230559384-d92513bc-eb9d-47ab-b8c0-88a0df5a1983.png)

**Comment by deakjahn** [comment]
Good catch but I can now find a few other similar cases where the variable extraction gives a similar error. I have to look into them all. (Basically, you can't extract some kind of expressions.)
